include/variables.h

```diff
@@ -168,10 +168,12 @@ extern u8 gSampleBankTable[];
 
 extern SaveContext gSaveContext;
 extern GameInfo* gGameInfo;
-extern u16 D_8015FCC0;
-extern u16 D_8015FCC2;
-extern u16 D_8015FCC4;
-extern u8 D_8015FCC8;
+
+extern u16 gCamAtPointsAppliedFrame;
+extern u16 gCamEyeAppliedFrame;
+extern u16 gCamAtAppliedFrame;
+extern u8 gUseCutsceneCam;
```

Comment:
possibly, though really they should be internal. they are only exposed because a super jank actor decided to copy paste some parts of the internal z_demo code for pretty much no reason

Reply:
and REALLY these data variables should have just been in the cs context like this one here. its very strange design
```
   /* 0x18 */ u16 camEyePointsAppliedFrame; // when set, indicates that the cam `eye` data was applied
```
but, I think I value keeping the naming similar across these (other than the extra `g`) then making it seem like a variable that has significant meaning outside of the system, when it doesnt

Reply:
id go as far as to say that, eventually when headers are split, this variable should be externed ONLY in bgtreemouth, as opposed to externing it in the header and making it seem like it should be used externally. id even want it named with an `s` prefix, with this one usage being seen as a hack. but could understand if others werent on board with that

Reply:
I think I will leave the names as-is and this can be revisited when z_demo has its own header.
at that point, the conversation of whether or not to extern this variable out to the public will be raised, and I can bring up my above points

Reply:
Fine with me, seems to be a special case anyway so as long as we keep it in mind